discord/embeds.py

```diff
@@ -422,6 +422,37 @@ def add_field(self, *, name, value, inline=True):
             self._fields = [field]
 
         return self
+   
+    def insert_field_at(self, index, *, name, value, inline=True):
+        """Inserts a field before a specified index to the embed.
+        
+        This function returns the class instance to allow for fluent-style
+        chaining.
+        
```

Comment:
Needs a `.. versionadded:: 1.2.0` somewhere in here.

Reply:
Done